### DISCUSSION TITLE
check-in-speaker & listener 앱 이름 공모

### DISCUSSION THREAD

**Discussion by Jsueeee** [euphony-io/euphony#102] [Ideas]
check-in-speaker & listener 앱의 새로운 이름을 지어주세요! 😁😁

**Comment by judemin** [comment]
check-in하고 euphony니까

Check-Eu (체크유) 어떨까요!?

**Reply by judemin** [reply]
절망적인 작명센스 죄송합니다

**Reply by aiclaudev** [reply]
체크유ㅋㅋㅋㅋ
Check you라는 뜻도있고, 사투리같아서 구수하고 좋은데요??

**Reply by dlwlstks96** [reply]
첫 아이디어부터 커트라인이 높네요...👍👍

**Reply by seohy02** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ절망적이라니 ,,,,  구수하고 친근하고 좋습니닿👍👍👍

**Reply by Jsueeee** [reply]
오 체크유 ㅋㅋㅋㅋㅋ 좋은데요?? 스피커와 리스너를 구분할 수 있는 네이밍 아이디어가 있다면 추가로 부탁드려요 ㅎㅎ

**Reply by jopopcorn** [reply]
체크유 귀엽네요ㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by hoho00** [reply]
저도 Check eu 생각했었는데 이미 훨씬 빠른 사람이 있었네요 좋은 것 같아요
체크유~

**Comment by kuro11pow2** [comment]
wave pass

생각나는대로 적어봤습니다... 🤔

**Reply by kuro11pow2** [reply]
정부에서 미는(?) 전자출입명부가 KI-PASS 인데 어떤 약자인지를 모르겠네요 검색해도 안 나오구 ㅋㅋ;
http://kipass.co.kr/
https://news.seoul.go.kr/welfare/files/2020/08/5f4b8da61fa673.12654480.pdf

**Reply by aiclaudev** [reply]
http://129.go.kr/faq/faq01_view.jsp?n=5638
저도 궁금해서 찾아봤는데, Korea Internet PASS 라고 하네요!!

**Reply by dlwlstks96** [reply]
아이디어 뱅크이십니다!!!!🎉🎉

**Reply by seohy02** [reply]
오!!! 뭔가 "PASS"가 들어가니까  전문적(?)으로 보이는 느낌이 있는 것 같아요 ㅎㅎㅎ👍👍👍👍

**Reply by YoungSeokHong** [reply]
wave pass 좋네요! 직관적이고😀

**Reply by designe** [reply]
오 저두 웨이브패스 괜찮은데여?!

**Reply by jopopcorn** [reply]
웨이브패스 괜찮아요!!

**Reply by HACore** [reply]
웨이브패스 멋진데요..👍

**Reply by eunzinn** [reply]
웨이브패스 간결하고 좋아요!

**Reply by judemin** [reply]
웨이브 패스 직관적이고 좋아요!!

**Reply by kuro11pow2** [reply]
acoustic pass

**Reply by kuro11pow2** [reply]
acoustic check-in

**Reply by kuro11pow2** [reply]
A-PASS

**Reply by kuro11pow2** [reply]
wave check-in

**Reply by kuro11pow2** [reply]
W-PASS

**Comment by aiclaudev** [comment]
이지인(easy-in) 

상용화가 된다면 큐알코드보다 더 편리할 것 같아서요ㅎㅎ!

**Reply by seohy02** [reply]
장점이 확 부각되는 이름인 것 같아요!! 뭔가 입에도 착착 붙네욤👍👍👍👍

**Reply by kuro11pow2** [reply]
어감도 좋고 편안한 느낌이라 좋네요!!

**Comment by designe** [comment]
I-hate-QR-check-in ?!
ㅋㅋㅋㅋㅋㅋㅋ

**Reply by Jsueeee** [reply]
저 이거 좋아욬ㅋㅋㅋㅋㅋㅋㅋ 짧게 I hate QR 도 괜찮은 것 같습니다 ㅋㅋㅋㅋㅋㅋㅋ

**Reply by kuro11pow2** [reply]
엄청 직관적이네요 ㅋㅋㅋㅋ

**Reply by HACore** [reply]
보면 뇌리에 강렬하게 박힐 이름이네요ㅋㅋㅋ

**Comment by YoungSeokHong** [comment]
안심소리 방명록

조금 정부에서 만든 것 처럼 네이밍 해봤습니다 ㅋㅋㅋㅋㅋ

**Reply by kuro11pow2** [reply]
ㅋㅋㅋ 정부 공식 앱 K-안심소리 방명록

**Reply by jopopcorn** [reply]
귀엽고 좋은데요?!

**Reply by HACore** [reply]
진짜 정부공식앱같아요ㅋㅋㅋ

**Comment by ycs1m1yk** [comment]
1. euphony는 18khz 이상의 불가청주파수 대역을 활용합니다.
2. 인간의 가청주파수 이상의 높은 주파수대역에서 소통하는 동물하면 돌고래가 먼저 떠오릅니다.
    **-> Dolphin**
3. 기존 QR체크인, 수기작성, 안심콜등과 달리 음파통신으로 체크인하면 마치 드라이브스루처럼 지나가며 체크인을 할 수 있을것입니다. **-> Thru**

1~3번을 종합하여 check-in-speaker & listener 앱 이름으로 **Dolphin Thru** 를 제안합니다.

**Reply by designe** [reply]
안심돌고래?!

**Reply by kuro11pow2** [reply]
오 돌핀쓰루는 멋지고 안심돌고래는 귀여운데요 ㅋㅋㅋㅋ

**Reply by judemin** [reply]
안심돌고래 귀여워서 좋아요!

**Reply by ycs1m1yk** [reply]
안심돌고래가 더 맘에드네요 🤣

**Reply by designe** [reply]
K-안심돌고래

**Comment by seohy02** [comment]
K-돌핀

**Reply by kuro11pow2** [reply]
ㅋㅋㅋㅋㅋㅋ

**Reply by judemin** [reply]
ㅋㅌㅋㅋㅋㅋㅋㅋㅋ

**Reply by designe** [reply]
돌핀K

**Reply by yeonns** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by ycs1m1yk** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by jopopcorn** [comment]
소입(소리없는 입장확인)

남녀노소 쓰는 앱이기 때문에 한글 이름으로 생각해봤습니다 ㅎㅎ

**Reply by Jsueeee** [reply]
한글이름 부르기 쉬워서 좋은 것 같아요!!

**Reply by hoho00** [reply]
뭔가 역설적인 네이밍이라서 좋은 것 같아요. 음파를 통해 데이터를 전송하지만, 소리 없는 입장. 좋은 이름이네요

**Comment by HACore** [comment]
저는 지금까지 나온 의견을 종합해
```
wave-in
```
밀어봅니다ㅋㅋㅋ

**Reply by ycs1m1yk** [reply]
와 완전체네요ㅋㅋㅋ

**Reply by YoungSeokHong** [reply]
여기서 생각해냈는데 wave-in해서 wavin이라 읽는 것도 좋겠네요!

**Reply by HACore** [reply]
Dolphin wave-in

**Reply by psh2849** [reply]
> 여기서 생각해냈는데 wave-in해서 wavin이라 읽는 것도 좋겠네요!

오.. 👍

**Reply by dlwlstks96** [reply]
좋은 아이디어입니다!!👍👍👍👍

**Comment by ycs1m1yk** [comment]
`두둥등장`

설명은 생략합니다

**Reply by judemin** [reply]
ㄷㄷㄷㅈ

**Reply by jopopcorn** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by hoho00** [reply]
두둥등장 좋은데요?

**Reply by HACore** [reply]
ㄷㄷㄷㅈ

**Comment by judemin** [comment]
모두의 체크인 해서
```
Uni-Check (유니체크)
```
어떨까요?

**Comment by ycs1m1yk** [comment]
Euphony로 한번에 체크인

`Eu-in-one` or `U-in-one` or `Eu인원` or `유인원`

**Reply by seohy02** [reply]
유인원ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by jopopcorn** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ유인원 재밌네요

**Reply by kuro11pow2** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by HACore** [reply]
유인원ㅋㅋㅋㅋㅋㅋㅋ

**Reply by judemin** [reply]
유인원 좋네요 ㅋㅌㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by Jsueeee** [reply]
유인원이요?

**Reply by Jsueeee** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by aiclaudev** [reply]
ㅋㅋㅋㅋㅋㅋ역시 아이디어뱅크십니다 ㅋㅋㅋㅋ

**Reply by Jsueeee** [reply]
저 유인원 밀어봅니다 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by hoho00** [reply]
앱 이름 유인원 해서 스플래쉬 이미지에 원숭이 로고화 시켜서 넣으면 재밌을 것 같아요 ㅋㅋㅋ

**Comment by jopopcorn** [comment]
심플하게 체크체크 하고 줄여서 쳌쳌 ... 😅

**Comment by psh2849** [comment]
바코드 찍듯 체크인 할 수 있으니 beep (삡..)

**Reply by HACore** [reply]
오 비프음도 떠오르고 귀여운 네이밍이네요ㅎㅎ

**Reply by jopopcorn** [reply]
ㅋㅋㅋㅋㅋㅋㅋ귀엽네요!!

**Comment by kuro11pow2** [comment]
drive-thru 처럼  wave-thru

**Comment by hoho00** [comment]
```유고걸``` 
```입장유``` 
```WUA``` : Where you at? 우아

**Comment by sujin9** [comment]
한글이름 생각해보니 갑자기 떠오른 이름입니다!
`누구왔소` -매장용 `내가왔소` -방문자용

**Reply by jopopcorn** [reply]
오 귀여워요!!!

**Reply by hoho00** [reply]
식당이나 매장 들어가서 "내가왔소" 해주세요 하면 재밌을 것 같아요.

**Comment by judemin** [comment]
Dolphin이랑 엮어서
```
DolPha (돌파)
```

**Reply by judemin** [reply]
민상연 완료하였습니다!

**Reply by kuro11pow2** [reply]
오 멋있네요!! 근데 문득 떠오른 돌파... 감염

**Reply by judemin** [reply]
> 오 멋있네요!! 근데 문득 떠오른 돌파... 감염

아니 이 뜻은 아니었는데 ㅋㅌㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by ycs1m1yk** [reply]
돌파감염ㅜㅜ

**Reply by YoungSeokHong** [reply]
돌파감염 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅠㅠ

**Reply by aiclaudev** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by aiclaudev** [reply]
돌파감염 재밌네요 ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by eunzinn** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by YoungSeokHong** [comment]
쉽게 들어가고 (안심번호로) 안전하게 나온다 해서
Simple In Safe Out
```SISO```

**Reply by judemin** [reply]
시소 귀엽네요 ㅎㅎ

**Reply by ycs1m1yk** [reply]
깔끔하고 좋아요!

**Comment by Jsueeee** [comment]
K-돌핀이 맘에 들어서
돌핀패스...?

**Reply by Jsueeee** [reply]
근데 저는 유인원이 좋습니다...

**Comment by seohy02** [comment]
유미패스
Eu meet pass
유미의세포들을보고 ..

**Comment by kuro11pow2** [comment]
Eurang팀 여기로 오세여

**Reply by aiclaudev** [reply]
뇌정지가 왔습니다...

**Reply by ycs1m1yk** [reply]
왔습니다!

**Reply by judemin** [reply]
완료했습니다!

**Reply by aiclaudev** [reply]
다 했습니다!!

**Reply by seohy02** [reply]
넴~

**Comment by aiclaudev** [comment]
렛유인

**Comment by aiclaudev** [comment]
어서오세Eu

**Comment by eunzinn** [comment]
Soundpass?
생각이 안나네요 ㅠㅠ

**Comment by HACore** [comment]
사일런스 패스

**Reply by jopopcorn** [reply]
줄이면 싸패..🤣

**Reply by HACore** [reply]
> 줄이면 싸패..🤣

으악ㅋㅋㅋㅋㅋㅠ

**Reply by Jsueeee** [reply]
세상에

**Reply by jopopcorn** [reply]
죄삼다......... 😅

**Reply by Jsueeee** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by YoungSeokHong** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by designe** [comment]
마지막 선택

**Reply by designe** [reply]
wave pass

**Reply by designe** [reply]
안심돌고래

**Reply by designe** [reply]
wave-in

**Reply by designe** [reply]
누구왔소/내가왔소

**Reply by designe** [reply]
SISO

**Reply by ycs1m1yk** [reply]
유인원 셀프추가해봅니다
